index.js

```diff
@@ -40,3 +40,8 @@ module.exports.butWhatActuallyIsCinnamonToastCrunchThough =
       "just might be a cereal involving cinnamon, toast, and something " +
       "crunchy!"
   }
+
+module.exports.butWhyIsCinnamonToastCrunchGood = 
+  function nobodyEvahExplainedItToMeToo () {
+    return "It's simply a fact of life, you know? Birds fly, gravity falls, stevens universe, Cinnamon Toast Crunch is good."
```

Comment:
Oh! An extra addition: Let's match the line breaking of `butWhatActuallyIsCinnamonToastCrunchThough` too:

```suggestion
    return "It's simply a fact of life, you know? Birds fly, gravity falls, " +
      "stevens universe, Cinnamon Toast Crunch is good."
```